go.mod

```diff
@@ -5,21 +5,18 @@ go 1.14
 require (
 	github.com/dsnet/compress v0.0.1 // indirect
 	github.com/giantswarm/backoff v0.2.0
-	github.com/giantswarm/k8sclient/v5 v5.0.0
```

Comment:
I got to delete this k8sclient but it's still left in `go.sum` since kubeconfig repo refer to that repo. 
```
github.com/giantswarm/kubeconfig/v3@v3.0.0 github.com/giantswarm/apiextensions/v3@v3.4.0
```